### PR TITLE
Add coq-dpdgraph ci

### PR DESCRIPTION
**Pull request by JasonGross**
Also rearrange Makefile.ci so that subsequent ci PRs won't conflict as much

### CODE REVIEW

File: dev/ci/ci-coq-dpdgraph.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+source ${ci_dir}/ci-common.sh
+
+coq_dpdgraph_CI_DIR=${CI_BUILD_DIR}/fiat-crypto
```
Comment (Zimmi48) [inline dev/ci/ci-coq-dpdgraph.sh]: Wrong copy-paste here.

File: Makefile.ci [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+    ci-geocoq \
+    ci-hott \
+    ci-iris-coq \
+    ci-math-classes \
+    ci-math-comp \
+    ci-metacoq \
+    ci-sf \
+    ci-tlc \
+    ci-unimath \
+    ci-vst
```
Comment (Zimmi48) [inline Makefile.ci]: I am strongly supporting this change.

**Comment by ejgallego** [comment]
Oh, I'd say please let's target 8.7, otherwise in our current model we may not be able to merge back if the plugin is not ported.

**Comment by ejgallego** [comment]
Also please put the contribution in the `allow-fail` category for now, the default policy is that new devs will undergo a quarantine period.

**Comment by JasonGross** [comment]
How about putting it in allow-fail and targeting 8.6; this will allow it to be merged, and we can see if it works with trunk, and only move it out of allow-fail if it does?

**Comment by ejgallego** [comment]
Which version does their master branch target?

**Comment by ejgallego** [comment]
I will release the first draft of the Travis/CI policy hopefully this week; all that things should become clear. No point in putting it in 8.6 only.

**Comment by ejgallego** [comment]
Also note that with 8.7 being frozen today, 8.6 is not in the CI radar anymore.

**Comment by maximedenes** [comment]
> today

tomorrow

**Comment by JasonGross** [comment]
> Which version does their master branch target?

8.6, I believe

**Comment by ejgallego** [comment]
You had ported it to 8.7 right? If so, we could submit a PR and see if they are ok having their master branch to track trunk, and their 8.7/8.6 branch to track the corresponding Coq releases.

**Comment by JasonGross** [comment]
No, I haven't ported it to 8.7 yet :-/

**Comment by ejgallego** [comment]
Ah OK, I was under the impression that you had an 8.7 branch. Delaying this PR until coq-dpgraph is ready for 8.7.

**Comment by JasonGross** [comment]
I've pushed an overlay that I hope will make it build, but see also https://github.com/Karmaki/coq-dpdgraph/issues/18

**Comment by ejgallego** [comment]
Great, keeping an eye here https://travis-ci.org/JasonGross/coq/builds/238962757

**Review by ejgallego — APPROVED** [review]

**Comment by ejgallego** [comment]
Ok, this is ready to go in.

**Comment by JasonGross** [comment]
Any update on this?

**Pull request merged**